internal/app/tfsec/formatters/default.go

```diff
@@ -19,10 +19,11 @@ import (
 )
 
 var severityFormat = map[severity.Severity]string{
-	severity.Low:    tml.Sprintf("<white>%s</white>", severity.Low),
-	severity.Medium: tml.Sprintf("<yellow>%s</yellow>", severity.Medium),
-	severity.High:   tml.Sprintf("<red>%s</red>", severity.High),
-	"":              tml.Sprintf("<white>%s</white>", severity.Low),
+	severity.Low:      tml.Sprintf("<white>%s</white>", severity.Low),
+	severity.Medium:   tml.Sprintf("<yellow>%s</yellow>", severity.Medium),
+	severity.High:     tml.Sprintf("<red>%s</red>", severity.High),
+	severity.Critical: tml.Sprintf("<red>%s</red>", severity.Critical),
```

Comment:
Is `BrightRed` an option?

Reply:
There is `lightred`, so i'll flip them round :+1:

Reply:
Nice

Reply:
Used bold instead, worked better